easybuild/easyconfigs/x/XQilla/XQilla-2.3.0-foss-2015a-Xerces-C++-3.1.2.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'XQilla'
+version = '2.3.0'
+
+homepage = 'http://xqilla.sourceforge.net/HomePage'
+
+description = """XQilla is an XQuery and XPath 2 library and command line
+utility written in C++, implemented on top of the Xerces-C∞ library."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [
+    'http://downloads.sourceforge.net/project/xqilla/xqilla/%s' % version,
+]
+
+configopts = '--with-xerces=$EBROOTXERCESMINCPLUSPLUS'
+
+xerces_version = '3.1.2'
+versionsuffix = '-Xerces-C++-%s' % xerces_version
```

Comment:
OK, I se there are easyconfigs already for XQilla 2.3.0, I understand you don't want to mess with those

but why not add easyconfigs for XQilla 2.3.2 here instead, which include Xerces as a dep? it should be fine to do that without using a versionsuffix, no?